src/fmeval/constants.py

```diff
@@ -125,6 +125,8 @@ class DatasetColumns(Enum):
 JUMPSTART_MODEL_VERSION = "jumpstart_model_version"
 MODEL_ID = "model_id"
 SPEC_KEY = "spec_key"
+INFERENCE_CONFIG_COMPONENTS = "inference_config_components"
+TGI = "tgi"
```

Comment:
@evakravi what's the difference between "tgi" and "lmi"? The code works with either one of them, and it's not clear from the JumpStart code what these represent.